File: 1.two-sum.java
// @leet start
class Solution {
    public int[] twoSum(int[] nums, int target) {
		Map<Integer, Integer> map = new HashMap<>();
		for (int i =0; i< nums.length; i++) {
			map.put(nums[i],i);
		}
		for (int i = 0; i < nums.length; i++) {
			int seek = target- nums[i];
			if (map.containsKey(seek) && map.get(seek)!=i ){
				return new int[]{map.get(seek), i};
				
			}
	
		}
		return new int[0];

    }
}
// @leet end

File: 1.two-sum.go
// @leet start
func twoSum(nums []int, target int) []int {
	for i := 0; i < len(nums); i++ {
		for n := i+1; n < len(nums); n++ {
			if nums[i]+nums[n]==target {
				return []int{i,n} 
			}
		}
	}
		return[]int{}
}
// @leet end
